packages/ui5-middleware-livetranspile/README.md

```diff
@@ -17,6 +17,9 @@ verbose logging
 array of paths inside `$yourapp/webapp/` to exclude from live transpilation,  
 e.g. 3-rd party libs in `lib/*`
 
+- transpileAsync: `true|false`
+transpiling `async/await` using [this Babel plugin](https://www.npmjs.com/package/babel-plugin-transform-async-to-promises), which doesn't require the regenerator runtime ([Issue #242](https://github.com/petermuessig/ui5-ecosystem-showcase/issues/242))
```

Comment:
could you please add 3 spaces before "using" so that a line break is displayed?

Reply:
I'm not sure if I misunderstand you. Wouldn't it be more consistent to add the spaces after line 20, and then somewhere in the paragraph of line 21?

Also, out of interest, why are the line breaks added manually?